static/templates/settings/bot-settings.handlebars

```diff
@@ -84,11 +84,11 @@
     <div id="get_api_key_box" class="side-padded-container">
       {{#tr this}}
       <p>Please re-enter your password to confirm your identity.
-        (<a href="/accounts/password/reset/" target="_blank">Forgotten it?</a>)</p>
+        <a href="/accounts/password/reset/" target="_blank">{{t "Forgotten it?" }}</a></p>
       {{/tr}}
       <form action="/json/fetch_api_key" method="post" class="form-horizontal">
         <div class="control-group">
-          <label for="password" class="control-label">{{t "Current password" }}</label>
+          <label for="password" class="control-label">{{t "Current password" }}</label>&nbsp;
```

Comment:
> what's the story with this piece?

To fix this, basically.

![Label without space](https://cloud.githubusercontent.com/assets/7356565/24317187/305d61fe-10f5-11e7-8320-8995899d1a29.png)

But I just noticed we already had some CSS to fix that, so I'm updating it.

---

About the separate commits, when is it okay to do so?

I know that from the logging perspective is much clearer, but sometimes I'm a bit reluctant because I don't want to create commits/PRs with minimal changes.  
For instance, if I had kept this `&nbsp;` thing in a different commit, it would have probably required a new PR, since it has nothing to do with i18n.